static/styles/settings.css

```diff
@@ -1386,13 +1386,23 @@ input[type="checkbox"] {
         }
     }
 
-    .sidebar {
+    .sidebar-wrapper {
         float: left;
         position: relative;
         width: 250px;
         height: 100%;
+
+        .tab-container {
+            box-sizing: border-box;
+            height: 45px;
```

Comment:
Added CSS for tab switcher as it was moved outside the simplebar, this matches it with the settings header for consistent styling.